components/page-heading.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
```

Comment:
Do you mind changing this `div` element to a `header`? It will make the page a little more accessible and search engines can do a better job interpreting this part of a page as a title as well.